Lib/ufo2ft/featureWriters/markFeatureWriter.py

```diff
@@ -59,6 +78,26 @@ class MarkToLigaPos(AbstractMarkPos):
 
     Statement = ast.MarkLigPosStatement
 
+    def warnIfAmbiguous(self, log):
```

Comment:
the two `warnIfAmbiguous` methods look almost identical, they could be probably factored out and reused

Reply:
Done